go-controller/pkg/allocator/udn/vrf.go

```diff
@@ -0,0 +1,16 @@
+package udn
+
+import (
+	idallocator "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/allocator/id"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/config"
+)
+
+const (
+	vrfTableIDName = "vrf-table"
+)
+
+// AllocateVRFTable will calculate the table ID to use with the created VRF
+// devices to implement local gateway with user defined networks.
+func AllocateVRFTable(vrfInterfaceIndex int) (uint, error) {
+	return idallocator.CalculateID(vrfTableIDName, uint(vrfInterfaceIndex), config.UserDefinedNetworks.VRFTableBase, config.UserDefinedNetworks.MaxNetworks)
```

Comment:
isn't this just ifaceIndex+baseval always? then if it exceeds 30K for UDNs and 60K for EIPs you return the error right?
what is the max value for ifaceIndex?